core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -88,7 +88,7 @@ UNWIND(	.fnstart)
 	 * PCR
 	 * - no change latency, enable clk gating
 	 */
-	mov_imm r0, 0x00004000
+	mov_imm r0, 0x00004800
```

Comment:
OK, it seems those bit should be set here, but they should be bitwise or:ed in to avoid changing other bits.